vcx/libvcx/src/api/vcx.rs

```diff
@@ -153,6 +153,19 @@ fn _finish_init(command_handle: CommandHandle, cb: extern fn(xcommand_handle: Co
                 cb(command_handle, e.into());
             }
         }
+
+        match settings::get_config_value(settings::CONFIG_WEBHOOK_URL) {
+            Ok(webhook_url) => match ::messages::agent_utils::update_agent_webhook(&webhook_url) {
+                Ok(()) => debug!("Agent webhook url updated on init"),
+                Err(e) => {
+                    error!("Error updating agent webhook on init: {}", e);
+                    cb(command_handle, e.into());
+                    return Ok(());
+                }
+            }
+            Err(e) => debug!("Error reading agent webhook from settings: {}", e)
```

Comment:
- If `webhook_url` is not present in the `agent provision` passed to `vcx_init`, `get_config_value` will return Error. So we should just in info/debug that webhook url will not be updated since it was not provided. 
- this match branch should also call success callback `cb(command_handle, e.into());`